theSrc/scripts/LabeledScatter.js

```diff
@@ -220,21 +222,21 @@ class LabeledScatter {
     }
   }
 
-  getLegendPointsRect (plotly_chart_layout, is_legend_points_to_right_of_plotly_legend, nsewdrag_rect, config) {
-    if (is_legend_points_to_right_of_plotly_legend) {
+  getLegendPointsRect (plotly_chart_layout, is_legend_elements_to_right_of_plotly_legend, nsewdrag_rect, config) {
+    if (is_legend_elements_to_right_of_plotly_legend) {
       return {
         x: nsewdrag_rect.width + this.plotlyLegendOrColorBarWidth(),
         y: LEGEND_POINTS_PADDING_TOP,
```

Comment:
What happens here if the legend is partially on the plot area? e.g. x = 0.8?

Reply:
The legend points are drawn assuming the plotly legend is not there (see isPlotlyLegendOnRight). It may overlap with the legend in some cases which is not ideal but I think it is OK.

Reply:
I have made changes so that the legend points and bubble legend will be drawn to the right of the plotly legend when the plotly legend is partially in the plot area (see the 8th legend position test snapshot). There should be no overlap.